examples/leds_shell/Makefile

```diff
@@ -0,0 +1,52 @@
+####
+#### Sample Makefile for building applications with the RIOT OS
+####
+#### The example file system layout is:
+#### ./application Makefile
+#### ../../RIOT
+####
+
+# Set the name of your application:
+APPLICATION = leds_shell
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../
+
+# Uncomment this to enable scheduler statistics for ps:
+#CFLAGS += -DSCHEDSTATISTICS
+
+# If you want to use native with valgrind, you should recompile native
+# with the target all-valgrind instead of all:
+# make -B clean all-valgrind
```

Comment:
Same here. I think it makes sense to keep the examples consistent in what information they offer in the Makefile. Otherwise, who would have thought that specifically `examples/leds_shell` is the folder to look into for more detailed Makefile hints?